brushfire-tree/src/main/scala/com/stripe/brushfire/Error.scala

```diff
@@ -27,13 +28,10 @@ trait FrequencyError[L, M, E] extends Error[Map[L, M], Map[L, Double], E] {
 
   def monoid: Monoid[E]
 
-  def create(actual: Map[L, M], predicted: Map[L, Double]) = {
-    if (predicted.isEmpty)
-      monoid.zero
-    else {
-      monoid.sum(actual.map { case (label, count) => error(label, count, predicted) })
-    }
-  }
+  def create(actual: Map[L, M], predicted: Map[L, Double]): E =
+    monoid.sum(actual.iterator.map { case (label, count) =>
```

Comment:
you're removing the protection of the `predicted.isEmpty` case, which means the `maxBy` in `error` is going to throw if that's true, right?